lua/telescope/_extensions/neorg/backlinks/file_backlinks.lua

```diff
@@ -0,0 +1,34 @@
+local neorg_loaded, neorg = pcall(require, "neorg.core")
+
+assert(neorg_loaded, "Neorg is not loaded - please make sure to load Neorg first")
+
+local file_backlink_regex = require("telescope._extensions.neorg.backlinks.utils")
```

Comment:
I really don't like this
either name the file correctly to what it is or return a table part of which is this regex
I'd really much prefer the second option
also in other file